src/SpacetimeDBClient.cs

```diff
@@ -64,15 +64,15 @@ public DbConnectionBuilder<DbConnection, Reducer> OnConnect(Action<DbConnection,
             return this;
         }
 
-        public DbConnectionBuilder<DbConnection, Reducer> OnConnectError(Action<WebSocketError?, string> cb)
+        public DbConnectionBuilder<DbConnection, Reducer> OnConnectError(Action<Exception> cb)
```

Comment:
It seems now we're only using Exceptions as string wrappers. Is this because we're introducing our own error message now?

I'd be in favor of keeping the error types coming from the dotnet library in the event signature, and adding our own message on the side.

Reply:
I updated this API to match @gefjon ‘s API stability proposal. Prior to this PR it was incorrectly implementing the proposal. We’re starting broad before we expose any implementation details in our API